scripts/ci-ai-model-serving/tests/vllm-image/Containerfile

```diff
@@ -0,0 +1,23 @@
+# https://developers.redhat.com/articles/2025/01/30/build-and-deploy-modelcar-container-openshift-ai
+#
+# podman build -t quay.io/microshift/ai-testing-model:vllm-granite-3b-code-base-2k .
+# podman push quay.io/microshift/ai-testing-model:vllm-granite-3b-code-base-2k
+
+FROM registry.access.redhat.com/ubi9/python-311:latest as base
+
+USER root
+
+RUN pip install huggingface-hub
+
+# Download the model file from hugging face
+WORKDIR /tmp
```

Comment:
Is the workdir change necessary? The model seems to be downloaded to /model

Reply:
hadolint complaint: 
```
 ./scripts/ci-ai-model-serving/tests/vllm-image/Containerfile:13 DL3045 warning: `COPY` to a relative destination without `WORKDIR` set. 
```

So it was either `WORKDIR /tmp` or
```
COPY download_model.py /tmp
RUN python /tmp/download_model.py
```
but I figured that `WORKDIR` would work without retesting (laziness).
Do you prefer adding `/tmp` to COPY and RUN?